modules/navier_stokes/include/physics/WCNSFVScalarTransportPhysics.h

```diff
@@ -33,11 +39,22 @@ class WCNSFVScalarTransportPhysics final : public NavierStokesPhysicsBase,
   bool hasScalarEquations() const { return _has_scalar_equation; }
 
 protected:
+  void addFVKernels() override;
+  void addFVBCs() override;
+
+  /// Names of the passive scalar variables
+  std::vector<NonlinearVariableName> _passive_scalar_names;
+  /// A boolean to help compatibility with the old Modules/NavierStokesFV syntax
```

Comment:
Not just that but it's necessary for your two phase mixture model physics when we're not adding the phase transport equation right?

Reply:
added